src/Grid/FeatureGrid/FeatureGrid.jsx

```diff
@@ -102,29 +136,38 @@ export class FeatureGrid extends React.Component {
     zoomToExtent: PropTypes.bool,
 
     /**
-     * Whether rows should be selectable or not.
+     * Whether rows and features should be selectable or not.
      *
      * @type {Boolean}
      */
     selectableRows: PropTypes.bool,
```

Comment:
should we rename to selectable? now that it not only means rows?

Reply:
Fixed in 495d1e8.